README.md

```diff
@@ -14,13 +14,16 @@ UI for navigating and organizing OpenAI's ChatGPT conversations.
 
 ![Screenshot](static/screenshot.png)
 
+[Docker Hub Image](https://hub.docker.com/layers/abalmahfadi/chat-history-app/1/images/sha256:5048890aa4a2ab8bd935a59fa12df64b7c028948d3ccd1610b35feedd77fde17?uuid=3C4FE425-AC33-44B0-8EE4-A1C54D21A883)
+
+
 ## Setup
 
 Currently can only be installed locally. Requires Python 3.10+
 
 1. [Export ChatGPT history](https://help.openai.com/en/articles/7260999-how-do-i-export-my-chatgpt-history-and-data)
 2. Unzip the download, place `conversations.json` in the `data` folder
-3. `make install`
-4. `make run`
-5. Open http://127.0.0.1:8000 in your browser
+3. `docker build -t chat-history-app .`
```

Comment:
i think some people would like a local install option. i think we should keep both local and container variants.

Reply:
## Setup

Currently can only be installed locally. Requires Python 3.10+
Run Locally:
1. `make install`
2. `make run`
3. Open http://127.0.0.1:8000 in your browser
4. [Export ChatGPT history](https://help.openai.com/en/articles/7260999-how-do-i-export-my-chatgpt-history-and-data)
5. Unzip the download, upload `conversations.json`
6. *Optional* - copy `secrets.template.toml` to `data/secrets.toml` and update OpenAI API key, then restart the server. First run will take a while to create embeddings. 10MB JSON: ~30 min, ~$0.10 cost.


Build in Docker:
1. `docker build -t chat-history-app .`
2. `docker run -p 80:80 chat-history-app`
3. Open http://127.0.0.1:8000 in your browser
4. [Export ChatGPT history](https://help.openai.com/en/articles/7260999-how-do-i-export-my-chatgpt-history-and-data)
5. Unzip the download, upload `conversations.json`
6. *Optional* - copy `secrets.template.toml` to `data/secrets.toml` and update OpenAI API key, then restart the server. First run will take a while to create embeddings. 10MB JSON: ~30 min, ~$0.10 cost.


I make this change please review them if it is ok I will add them to the next pull request.